src/main/java/com/squareup/kotlinpoet/CodeWriter.kt

```diff
@@ -302,14 +292,11 @@ internal class CodeWriter constructor(
   }
 
   private fun emitLiteral(o: Any?) {
-    if (o is TypeSpec) {
-      o.emit(this, null)
-    } else if (o is AnnotationSpec) {
-      o.emit(this, inline = true, asParameter = true)
-    } else if (o is CodeBlock) {
-      emitCode(o)
-    } else {
-      emit(o.toString())
+    when (o) {
```

Comment:
Could've also been `= when(o) {`. Do you prefer braces when the return type is `Unit`?

Reply:
Not necessarily. I'm not sure why I didn't do it for this one. Will look later.